lib/res/parsers/junit.rb

```diff
@@ -47,12 +47,14 @@ def attach_suite(component)
           if node.value == 'testcase'
             suite[index] = Hash.new
             suite[index] = attach_cases(node)
-          else 
-            suite[index] = Hash.new
-            suite[index]['type'] = "JUnit::#{node.value}"
-            suite[index]['name'] = node.attributes[:name]
-            suite[index]['classname'] = node.attributes[:classname] if suite[index]['classname'] != nil
-            suite[index]['children'] = attach_suite(node)
+          else
+            if node.respond_to?(:attributes) and node.respond_to?(:value) 
```

Comment:
Comments are treated as nodes in `ox` gem ex: https://github.com/bbc/res/pull/105/files#diff-fec93ee2d339e43dcb97e15c88506464R2

So, the tests below does include it indirectly